net: lib: coap_server: common: net.coap.server.secure reports RAM overflow

**Describe the bug**
The test `tests/net/lib/coap_server/common/net.coap.server.secure` fails.

Observed for `nrf54l15dk/nrf54l15/cpuflpr`.

**To Reproduce**
Steps to reproduce the behavior:
1. go to your zephyr dir
2. call `./scripts/twister -s net.coap.server.secure -p nrf54l15dk/nrf54l15/cpuflpr -v --inline-logs --overflow-as-errors`
3. See console output with error

**Expected behavior**
Valid console output

**Impact**
Not clear

**Logs and console output**
```
/opt/zephyr-sdk/riscv64-zephyr-elf/bin/lib/gcc/riscv64-zephyr-elf/12.2.0/riscv64-zephyr-elf/bin/ld.bfd: zephyr/zephyr_pre0.elf section `text' will not fit in region `RAM'
/opt/zephyr-sdk/riscv64-zephyr-elf/bin/lib/gcc/riscv64-zephyr-elf/12.2.0/riscv64-zephyr-elf/bin/ld.bfd: region `RAM' overflowed by 109456 bytes
collect2: error: ld returned 1 exit status
ninja: build stopped: subcommand failed.
```

**Environment (please complete the following information):**
 - OS: Ubuntu 20.04.1 LTS
 - Toolchain Zephyr SDK 0.17.0
 - Commit SHA or Version used: `v4.1.0-2843-g9c9c08`

**Additional information:**
Probably introducedy by commit b596b13e7c48d026f99c053d63ff0034ce0633d7

--- a/tests/net/lib/coap_server/common/testcase.yaml
+++ b/tests/net/lib/coap_server/common/testcase.yaml
@@ -1,5 +1,7 @@
 common:
-  min_ram: 16
+  min_ram: 40
+  min_flash: 180
+  depends_on: netif
   tags:
     - net
     - coap

